packages/grid/_modules_/grid/models/api/gridApi.ts

```diff
@@ -37,7 +37,7 @@ export interface GridApi
     GridPaginationApi,
     GridCsvExportApi,
     GridFocusApi,
-    FilterApi,
+    GridFilterApi,
     ColumnMenuApi,
```

Comment:
**Side**

We have the same wrong naming here:

```suggestion
    GridColumnMenuApi,
```

Reply:
I have created https://github.com/mui-org/material-ui-x/issues/1705 so we don't forget. We can come back to it at some point in the future.